chain/src/txhashset/txhashset.rs

```diff
@@ -41,6 +41,7 @@ use std::time::{Instant, SystemTime, UNIX_EPOCH};
 
 const HEADERHASHSET_SUBDIR: &'static str = "header";
 const TXHASHSET_SUBDIR: &'static str = "txhashset";
+const TXHASHSET_SANDBOX_SUBDIR: &'static str = "txhashset_sandbox";
```

Comment:
See earlier comment above.

Proposal: The "sandbox" should be under $TMPDIR and not under in the regular `db_root`.
The _only_ files saved to disk under `db_root` should be validated.

Reply:
done with `env::temp` folder.